projects/ngx-qgrid-plugins/src/lib/cell-tooltip/cell-tooltip.component.ts

```diff
@@ -0,0 +1,71 @@
+import {
+	ChangeDetectorRef,
+	ChangeDetectionStrategy,
+	Component,
+	OnInit,
+	Input,
+	ApplicationRef,
+} from '@angular/core';
+import { DomTd, GridPlugin, TemplateHostService } from '@qgrid/ngx';
+
+@Component({
+	selector: 'q-grid-cell-tooltip',
+	templateUrl: './cell-tooltip.component.html',
+	providers: [GridPlugin, TemplateHostService],
+	changeDetection: ChangeDetectionStrategy.OnPush,
+})
+
+export class CellTooltipComponent implements OnInit {
+	@Input() showDelay: number;
+
+	context: { $implicit: DomTd } = {
+		$implicit: null,
+	};
+	cellElement: HTMLElement;
+
+	constructor(
+		private plugin: GridPlugin,
+		private cd: ChangeDetectorRef,
+		private appRef: ApplicationRef
+	) { }
+
+	ngOnInit() {
+		const { model, observe, table } = this.plugin;
+
+		observe(model.mouseChanged)
+			.subscribe((e) => {
+				const { target } = model.mouse();
+				if (e.hasChanges('target')) {
```

Comment:
why we don't use state === 'move' also, the target can be changed on click?
also u can use e.state.target instead of model.mouse()

Reply:
Yep, we can change it to 'move', but also we will need at least 'leave'. 
As I can see, we updating the target on the mouse down and up in the current design.
Okay, I will change it to e.state.target instead.